### PR TITLE
ci: update owners for /tools/size-tracking

### PR DESCRIPTION
**Pull request by josephperrott**

### CODE REVIEW

File: .github/CODEOWNERS [hunk 1]
```diff
@@ -915,6 +915,7 @@ testing/**                                                      @angular/fw-test
 /tools/npm/**                                                   @angular/dev-infra-framework
 /tools/npm_workspace/**                                         @angular/dev-infra-framework
 /tools/public_api_guard/**                                      @angular/dev-infra-framework
+/tools/size-tracking/**                                         @angular/dev-infra-framework
```
Comment (gkalpak) [inline .github/CODEOWNERS]: Nit: Having patterns alphabetically ordered helps when searching whether a specific file/directory is covered.

**Review by devversion — APPROVED** [review]
LGTM aside of @gkalpak's comment.

File: .github/CODEOWNERS (hunk 1 above)
Comment (josephperrott) [reply .github/CODEOWNERS]: Classic off by one error :smile:

**Review by gkalpak — APPROVED** [review]

**Review by kara — APPROVED** [review]
LGTM

**Comment by kara** [comment]
merge-assistance: global approval

**Pull request closed without merging**